test/RequestHandlerTest.cpp

```diff
@@ -16,42 +17,39 @@ class RequestHandlerTest : public ::testing::Test {};
 
 TEST(RequestHandlerTest,
      subscribe_consume_n_last_messages_throws_if_incorrect_arguments_test) {
-  auto KafkaConnection = std::make_unique<ConnectKafkaFake>(ConnectKafkaFake());
   UserArgumentStruct UserArguments;
-  RequestHandler NewRequestHandler(std::move(KafkaConnection), UserArguments,
-                                   updateSchemas(false));
+  UserArguments.ConsumerMode = true;
+  RequestHandler NewRequestHandler(UserArguments, updateSchemas(false), false);
```

Comment:
Putting this `false` in an appropriately named `const bool` in the anonymous namespace would help readability of the tests. I don't think the `SchemaPath` variable is used any more so you can remove that whilst you are there :)